app.go

```diff
@@ -404,16 +417,16 @@ func HandleAction(action interface{}, context *Context) error {
 	if len(vals) == 0 {
 		fmt.Fprintln(os.Stderr,
 			"DEPRECATED Action signature.  Must be `cli.ActionFunc`")
-		return nil
+		return err
```

Comment:
Aaah, nvm, I think I see now that this would end up being a real error if there is a panic.